geomstats/geometry/discrete_surfaces.py

```diff
@@ -67,6 +73,10 @@ def __init__(
             equip=equip,
         )
 
+    def new(self, equip=True):
+        """Create manifold with same parameters."""
```

Comment:
Need Parameters description. Esp since equip is actually not used?

Explain what "same" parameters mean in the docstring.

Reply:
`equip` is used. A mistake on my side. Fixed.